UnitConverter/ViewController.swift

```diff
@@ -29,6 +31,8 @@ class ViewController: UIViewController, UIPickerViewDelegate {
         if let row = savedRow  {
             return row
         } else {
+            // we have 200 total rows from -100 to +100 so to make middle value 0 selection
+            // we have to get half of total row counts i.e. 200 / 2 = 100
```

Comment:
There's value in this comment, but it can be more succinct. Example:
`Middle row index is half of the total number of rows.`